### ISSUE TITLE
Show timestamps/relative timing info to requests

### ISSUE THREAD

**Issue by pimterry** [httptoolkit/feedback#90]
In some cases, it's useful to be able to see exactly when one request happened relative to another, not just the simple order that's currently shown.

---

_Does this affect you too? Click the +:smiley: icon in the top right and add a :+1: on issues to help decide where HTTP Toolkit goes next, or [go take a look & vote on the most popular ideas so far](https://github.com/httptoolkit/feedback/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc)._

**Comment by catalinghita8** [comment]
Hi @pimterry and thanks for your fast response!

I was thinking to have some kind of basic toggle (maybe in the View menu or in the UI) that would allow the devs to see timestamps for each request. Something like this:

<img width="463" alt="Screenshot 2020-07-21 at 09 37 28" src="https://user-images.githubusercontent.com/30151847/88021481-34a32100-cb36-11ea-9108-aa70a1838ccb.png">

Obviously my editing skills are not that good so the timestamps should match the UI of the App itself, but I think it's a bit more clear what I was referring to.

**Comment by mivanov** [comment]
This is one of the first things I looked for. Columns for start time and duration, hidden by default is fine as it's not important for many use cases.

**Comment by tehaon** [comment]
agreed, started using the tool and trying to compare two values between requests but i need to know the duration between requests to figure out the numbers I want. The tool looks really nice though.
